core/base/inc/TString.h

```diff
@@ -782,16 +782,8 @@ namespace llvm {
 }
 
 namespace cling {
-   // cling ValuePrinting
-   class Value;
-   std::string printValue(const TString* const p, const TString* const u,
-                          const Value& V);
-   std::string printValue(const TSubString* const p, const TSubString* const u,
-                          const Value& V);
-   std::string printValue(const std::string* const p, const std::string* const u,
-                          const Value& V);
-   std::string printValue(const std::string_view* const p, const std::string_view* const u,
```

Comment:
Will do. I put it in TString.h again, as the required header is not included in RuntimePrintValue.h. Let me know if I should move it there. Thanks!